oaknational/Oak-Web-Application
# Feat/pupil 1058/multiple learning outcomes

### Job Story

**When** I view lesson intros,
**I want to see multiple learning outcomes**
**So that pupils can understand the lesson outcomes**

### Implementation
- added phonics_outcome to gql query
- render the phonics outcome if available on the overview page

## How to test

1. Go to https://deploy-preview-2927--oak-web-application.netlify.thenational.academy/pupils/beta/previews/lessons/what-is-that-cest-and-singular-indefinite-articles/overview
2. You should see two lesson outcomes
3. check other lessons on pupil to see that they are still rendered correctly
4. check other lessons on teacher to see that they are still rendered correctly

### ACs

- [x]  In a MFL lesson (new curriculum), a pupil can view two learning outcomes during the lesson introduction.
- [x] other lessons are still rendered
